htdocs/core/actions_addupdatedelete.inc.php

```diff
@@ -253,7 +253,10 @@
 	if (!$error) {
 		$result = $object->update($user);
 		if ($result > 0) {
-			$action = 'view';
+			$urltogo = $backtopage ? str_replace('__ID__', $result, $backtopage) : $backurlforlist;
```

Comment:
In common cases, we should not have to make a redirect after an update.
The $object should be already up to date (due to ->update), so output the page with action = view should be enough.
In which use case,do you need a redirect ?

Reply:
without the header redirect the $_POST is not cleared. That way a user can not refresh the page or move back and forward without issue

Reply:
For which element, which url update can you reproduce this ?

Reply:
/projet/card.php?id=13 -> Edit -> Cancel -> /projet/card.php?id=13 | OK
/projet/card.php?id=13 -> Edit -> Save -> /projet/card.php | NOK